### DISCUSSION TITLE
Generally slower than Chromium PWA implementation

### DISCUSSION THREAD

**Discussion by jacksongoode** [filips123/PWAsForFirefox#73] [General]
Having compared the two on Fedora 35, it seems like PWA's on Firefox are far slower on boot. Maybe this has to do with the cold start it must make into a new profile? Are there any ways of improving this?

**Comment by filips123** [comment]
It's probably because FirefoxPWA currently copies files from another directory and removes Firefox startup cache every time you launch a PWA. It should be better once I change this to only copy files when needed (e.g., when there is a new version, see #53).
